code/game/objects/effects/spawners/lootdrop.dm

```diff
@@ -30,6 +30,20 @@
 			loot_spawned++
 	return INITIALIZE_HINT_QDEL
 
+/obj/effect/spawner/lootdrop/donkpockets
```

Comment:
too many donk pockets. this could work in some maint rooms, but leave chef with the normal ones roundstart. 

what you COULD do is make a "variety" box with random or pseudo-random donk pockets (gondola pockets excluded)